decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -17,7 +17,7 @@
         <%= link_to_current_or_new_conversation_with(@proposal.author) %>
       <% end %>
     </div>
-    <% unless @proposal.withdrawn? or !@proposal.withdrawable_by?(current_user) %>
+    <% if @proposal.withdrawn? && @proposal.withdrawable_by?(current_user) %>
```

Comment:
This logic makes no sense, you need to remove the `@proposal.withdrawn?` part now that you're checking this on the `withdrawable_by` method:

`<% if @proposal.withdrawable_by?(current_user) %>`